autoblocks/_impl/prompts/autogenerate.py

```diff
@@ -66,19 +57,14 @@ def execution_context_class_name(self) -> str:
     def manager_class_name(self) -> str:
         return f"{self.title_case_id}PromptManager"
 
-    @property
-    def minor_version_enum_class_name(self) -> str:
-        return f"{self.title_case_id}MinorVersion"
-
 
 HEADER = """############################################################################
 # This file was generated automatically by Autoblocks. Do not edit directly.
 ############################################################################
 
-from enum import Enum
 from typing import Union  # noqa: F401
 
-import pydantic
+import pydantic  # noqa: F401
```

Comment:
Why do we need these noqa tags here?

Reply:
b/c pydantic might not be used if you don't use params, so if we don't noqa it our autogenerated file will fail flake8 unused imports rule. could probably add the imports retroactively instead so we don't need to add noqa to things